my-app/src/Components/App/App.js

```diff
@@ -0,0 +1,152 @@
+import '../../index.css';
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import api from '../../utils/WeatherApi';
+import Main from '../Main/Main';
+import Header from '../Header/Header';
+import Footer from '../Footer/Footer';
+import About from '../About/About';
+import WeatherSearch from '../WeatherSearch/WeatherSearch';
+
+export default function App() {
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [isWeather, setIsWeather] = React.useState(false);
+  const [isTitle, setIsTitle] = React.useState(true);
+  const [isWeatherData, setIsWeatherData] = React.useState([]);
+  const [alerts, setAlerts] = React.useState([]);
```

Comment:
` React.useState([]);`

The check list says: The initial value of state variables contains the correct data type.

in line 109 you set an `object` so here it should be also `{}` by default